build-bin/dist_to_gh_pages

```diff
@@ -15,26 +15,37 @@
 
```

Comment:
note renaming this file will break deploy. you can fix that by also changing build-bin/deploy with the name change.

In general, unless using an IDE like IntelliJ which detects this sort of thing for you, go back and do a recursive search any time you rename or delete some part of the build.